src/main/java/org/geysermc/resources/state/type/wall/BlockTypeMapper.java

```diff
@@ -0,0 +1,18 @@
+package org.geysermc.resources.state.type.wall;
+
+import org.geysermc.resources.Pair;
+import org.geysermc.resources.state.StateMapper;
+import org.geysermc.resources.state.StateRemapper;
+
+// Used north value as all walls have it
+@StateRemapper(value = "north", blockRegex = ".*_wall.?$")
+public class BlockTypeMapper extends StateMapper<String> {

```

Comment:
Needs to be named to WallValueMapper. Naming convention is `(block)(java state)Mapper`.

Reply:
Java doesn't have this state so I'll rename it to WallBlockTypeMapper

Reply:
Meant to say WallNorthMapper, will update manually.

Reply:
Actually nevermind, seems like this uses the north state to do something completely different, so this is fine.

Reply:
WallNorthMapper already exists, this mapper is to map wall block types, I only used north because all walls have it, the north value is never used